packages/viz/src/lib/popups/Popup.ts

```diff
@@ -55,15 +55,13 @@ export class Popup {
       this._parentElement = canvasElem.parentElement;
     }
 
-    // this._deckInstance.on('onViewStateChange', this._render.bind(this));
-    const { onViewStateChange } = this._deckInstance.props;
-
+    const { onAfterRender } = this._deckInstance.props;
     this._deckInstance.setProps({
-      onViewStateChange: (...args: any) => {
+      onAfterRender: (...args: any) => {
```

Comment:
It looks like we force a new render... How did you come into this solution? Any tip or comment from vis.gl repo maybe?

Reply:
Not really, I just found that event callback in the [doc of deckgl](https://deck.gl/#/documentation/deckgl-api-reference/deck) which says:

_"Called right after the canvas rerenders."_

By this way we have the last view and we can use it to unproject the coordinates to the new pixels location.